### ISSUE TITLE
Unknown property `color` for component/system `material`. 

### ISSUE THREAD

**Issue by levistep32** [aframevr/aframe#3766]
**Description:**
In the console, I get this:
```
core:schema:warn 
Unknown property `color` for component/system `material`. 
```
- A-Frame Version: 0.8.2
- Platform / Device: Laptop

I have a couple times that I mention color in material, but I don't know which one.

**Comment by ngokevin** [comment]
needs more info.

**Comment by levistep32** [comment]
About what?

**Comment by ngokevin** [comment]
Well, you don't provide any code or any leads. It's a warning that could come from anywhere. This is perhaps a StackOverflow support question to ask here https://stackoverflow.com/tags/aframe

**Comment by jgbarah** [comment]
It seems this is still happening with 1.6.0. The following code triggers the message

```
core:component:warn Unknown property `color` for component `material`.
```

 in Chrome 131.0.6778.264 (Official Build) built on Debian GNU/Linux trixie/sid (64-bit):

```html
<!DOCTYPE html>

<html>
  <head>
    <script src="https://aframe.io/releases/1.6.0/aframe.min.js"></script>
  </head>
  <body>
    <a-scene>
        <a-entity geometry="primitive: box" material="shader: flat; color: red"></a-entity>
    </a-scene>
  </body>
</html>
```

**Comment by mrxz** [comment]
@jgbarah This is fixed on master and will be in the soon to be released 1.7.0 (see https://github.com/aframevr/aframe/pull/5526)

**Comment by jgbarah** [comment]
Thanks! I just checked in 1.7.0 and, yes, it is fixed. Thanks a lot for the good work!
